Makefile.kokkos

```diff
@@ -279,6 +284,15 @@ else
   KOKKOS_INTERNAL_OPENMPTARGET_FLAG := -fopenmp -foffload=nvptx-none
 endif
 
+ifeq ($(KOKKOS_INTERNAL_USE_OPENACC), 1)
+# Set OpenACC flags.
+ifeq ($(KOKKOS_INTERNAL_COMPILER_NVHPC), 1)
+  KOKKOS_INTERNAL_OPENACC_FLAG := -acc
+else
+  $(error Makefile.kokkos: OpenACC is enabled but the compiler must be NVHPC (got version string $(KOKKOS_CXX_VERSION)))
+endif
+endif
```

Comment:
Indentation is wrong but I can live with it

Reply:
Done!